assets/panel/debugger.properties

```diff
@@ -490,17 +490,30 @@ editor.removeBreakpoint=Remove breakpoint
 
 # LOCALIZATION NOTE (editor.editBreakpoint): Editor gutter context menu item
 # for setting a breakpoint condition on a line.
-editor.editBreakpoint=Edit breakpoint
+editor.editBreakpoint=Edit condition
```

Comment:
Quick note: to change strings like this, you need to use a new string ID (and updated the comment accordingly).
https://developer.mozilla.org/en-US/docs/Making_String_Changes